File: Assignment 5/calculator.cpp
#include<iostream>
using namespace std;
class Calculator
{
    int n,num1,num2;
    int sum=0,sub=0,mult=1,div=0;
    public :Calculator()
    {
        cout<<"------CALCULATOR-------";

    };
    void accept()
    {
        cout<<"\n CHOOSE ANY OF THE OPTION YOU PREFER:";
        cout<<"\n 1.Addition 2.Substraction 3.Multiplication 4.Division \n";
        cin>>n;
        switch(n)
        {
            case 1:cout<<"\nWelcome to addition";
                  cout<<"\nEnter 2 no's to be added:\n";
                  cin>>num1>>num2;
                  sum=num1+num2;

                  cout<<"The sum of the numbers is :\t"<<sum;
            case 2:cout<<"\nWelcome to Substraction";
                  cout<<"\nEnter 2 no's to be substraction:\n";
                  cin>>num1>>num2;
                  sub=num1-num2;

                  cout<<"The substraction of the numbers is :\t"<<sub;
            case 3:cout<<"\nWelcome to Multiplication";
                  cout<<"\nEnter 2 no's to be Multiplication:\n";
                  cin>>num1>>num2;
                  mult=num1*num2;
                  cout<<"The multiplication of the numbers is :\t"<<mult;

            case 4:cout<<"\nWelcome to Division";
                  cout<<"\nEnter 2 no's to be division:\n";
                  cin>>num1>>num2;
                  div=num1%num2;
                  cout<<"The division of the numbers is :\t"<<div;

            default:
            cout<<"Invalid choice:";
        }
    }

};
int main()
{
    Calculator c1;
    c1.accept();
}

File: JAVA/Assignment2/Calculator.java
import java.util.Scanner;
class Calculator
{
	int n;
	int num1,num2;
	int sum=0,sub=0,mul=0,div=0;
	public static void main(String[] args)
	{
		int n;
	    int num1,num2;
	    int sum=0,sub=0,mul=0,div=0;
		Scanner sc=new Scanner(System.in);
		System.out.println("enter 2 no's");
		num1=sc.nextInt();
		num2=sc.nextInt();
		System.out.println("Choose your option");
		System.out.println("1.Addition");
		System.out.println("2.Substraction");
		System.out.println("3.Multiplication");
		System.out.println("4.Division");
		System.out.println("exit");
		n=sc.nextInt();
		switch(n)
		{
			case 1:System.out.println("1.Addition"); 
                   sum=num1+num2;
                   	System.out.println("Addition="+sum);
					break;
            case 2:System.out.println("2.Substraction"); 
                   sub=num1-num2;
                   	System.out.println("substraction="+sub);
					break;
            case 3:System.out.println("1.multiplication"); 
                   mul=num1*num2;
                   	System.out.println("multiplication="+mul);
					break;
            case 4:System.out.println("division"); 
                   div=num1/num2;
                   	System.out.println("quotient="+div);
					break;
            case 5:System.out.println("exit"); 
			        break;
            default:
               		System.out.println("invalid choice"); 
                     break;					
		}
	}
}
		
